homeassistant/components/doorbird/__init__.py

```diff
@@ -137,16 +94,126 @@ def _reset_device_favorites_handler(event):
             for favorite_id in favorites[favorite_type]:
```

Comment:
`if doorstation is None` should include a `return`

Reply:
Already on the list after `rachio` (needs some surgery to clean it up a bit first)

Reply:
added